### ISSUE TITLE
Review of 3.1 Introduction (of the Discovery section)

### ISSUE THREAD

**Issue by rdeltour** [w3c/epub-specs#911]
The [Introduction](https://rawgit.com/IDPF/epub-revision/master/build/a11y/accessibility.html#sec-disc-intro) section should be marked as _informative_.

> One downside to this model, however, is that specific details about the accessibility of a publication typically do not travel with it.

The above sounds maybe a bit negative, can be reworded along the lines of: "This model, however, requires that specific details about the accessibility of a publication can travel with it"

> can only convey to consumers what is present in the metadata.

s/the metadata/metadata

> That the content meets the accessibility requirements of this specification is often not enough information; the specific affordances made to meet the requirements are equally important to know when determining usability.

Sounds maybe a bit mouthful?

> Users need to be able to gauge the suitability of an EPUB Publication when they purchase, borrow or otherwise obtain it (e.g., will it be adequate for an educational setting)

I would move the parenthesis after "suitability of an EPUB Publication"

**Comment by mattgarrish** [comment]
Generally agree, but I've adjusted some slightly differently.
